pkgs/applications/misc/gphoto2/default.nix

```diff
@@ -3,11 +3,11 @@
 }:
 
 stdenv.mkDerivation rec {
-  name = "gphoto2-2.5.11";
+  name = "gphoto2-2.5.15";
 
   src = fetchurl {
     url = "mirror://sourceforge/gphoto/${name}.tar.bz2";
-    sha256 = "1sgr6rsvzzagcwhc8fxbnvz3k02wr2hab0vrbvcb04k5l3b48a1r";
+    sha256 = "0xsa12k5fz49v8y4h3zahzr427a3ylxaf0k7hybrkp43g4i1lmxf";
```

Comment:
The hash doesn't match, on [Hydra](https://hydra.nixos.org/build/70501543) and locally.  Any idea why it's wrong?

Reply:
E.g., upstream updating it in-place in the meantime?

Reply:
Something wrong with sourceforge

```
curl -LO http://downloads.sourceforge.net/gphoto/gphoto2-2.5.15.tar.bz2
cat gphoto2-2.5.15.tar.bz2:
```
```
<html><head>
<title>SourceForge</title>
<!-- <script src="/js/jquery.com/jquery-1.11.0.min.js"></script> -->
<script src="https://code.jquery.com/jquery-1.11.0.min.js"></script>
<script src="https://sourceforge.net/js/mirrors.js"></script>
<script src="/js/sf.js"></script>
<script>
var DR_loc = DR_parse_hash_url();
if (DR_loc) {
    DR_sf_main(DR_loc);
} else {
    window.location.href = 'https://sourceforge.net/home.html';
}
</script>
</head><body>
<noscript>
We're sorry -- the Sourceforge site is currently in Disaster Recovery mode, and currently requires
the use of javascript to function.  Please check back later.
</noscript>
</body></html>
```

Reply:
Now it seems OK and we got it into our binary cache, too.